worlds/zork_grand_inquisitor/world.py

```diff
@@ -0,0 +1,210 @@
+            regions_locations: Set[ZorkGrandInquisitorLocations] = region_locations_mapping[region_enum_item]
+
+            # Locations
+            location_enum_item: ZorkGrandInquisitorLocations
+            for location_enum_item in regions_locations:
+                data: ZorkGrandInquisitorLocationData = location_data[location_enum_item]
+
+                location: ZorkGrandInquisitorLocation = ZorkGrandInquisitorLocation(
+                    self.player,
+                    location_enum_item.value,
+                    data.archipelago_id,
+                    region_mapping[data.region],
+                )
+
+                location.event = isinstance(location_enum_item, ZorkGrandInquisitorEvents)
+
+                if location.event:
+                    location.place_locked_item(
+                        ZorkGrandInquisitorItem(
+                            data.event_item_name,
+                            ItemClassification.progression,
+                            None,
+                            self.player,
+                        )
+                    )
+
+                location_access_rule: str = location_access_rule_for(location_enum_item, self.player)
+
+                if location_access_rule != "lambda state: True":
+                    location.access_rule = eval(location_access_rule)
+
+                region.locations.append(location)
+
+            # Connections
+            region_exit: ZorkGrandInquisitorRegions
+            for region_exit in region_data[region_enum_item].exits or tuple():
+                entrance_access_rule: str = entrance_access_rule_for(region_enum_item, region_exit, self.player)
+
+                if entrance_access_rule == "lambda state: True":
+                    region.connect(region_mapping[region_exit])
+                else:
+                    region.connect(region_mapping[region_exit], rule=eval(entrance_access_rule))
+
+            self.multiworld.regions.append(region)
+
+    def create_items(self) -> None:
+        quick_port_foozle: bool = self.options.quick_port_foozle.value == 1
+        start_with_hotspot_items: bool = self.options.start_with_hotspot_items.value == 1
+
+        item_pool: List[ZorkGrandInquisitorItem] = list()
+
+        item: ZorkGrandInquisitorItems
+        data: ZorkGrandInquisitorItemData
+        for item, data in item_data.items():
+            tags: Tuple[ZorkGrandInquisitorTags, ...] = data.tags or tuple()
+
+            if ZorkGrandInquisitorTags.FILLER in tags:
+                continue
+            elif ZorkGrandInquisitorTags.HOTSPOT in tags and start_with_hotspot_items:
+                continue
+
+            item_pool.append(self.create_item(item.value))
+
+        total_locations: int = len(self.multiworld.get_unfilled_locations(self.player))
+
+        for _ in range(total_locations - len(item_pool)):
+            item_pool.append(self.create_item(self.get_filler_item_name()))
```

Comment:
```suggestion
        item_pool += [self.create_filler() for _ in range(total_locations - len(item_pool)]
```

Reply:
Nice, didn't know there already was a helper for this. And this is a good candidate for a list comprehension too